cpp/powsybl-cpp/powsybl-api.h

```diff
@@ -314,6 +314,7 @@ typedef struct {
 typedef struct zone_struct {
     char* id;
     char** injections_ids;
+    int injections_ids_count;
```

Comment:
what is this field used for ?
I guess it is necessary for a size somewhere but it is not used in that way

Reply:
Ok I found it in the java method to create and delete the object